web/app/adapters/application.js

```diff
@@ -24,6 +24,11 @@ export default DS.JSONAPIAdapter.extend({
 export default DS.RESTAdapter.extend({
   namespace: 'rest',
   pathForType: function (type) {
-    return  Ember.String.pluralize(type);
+    if (type == "tunein") {
+      return type
```

Comment:
nit: semicolon missing. 
Further, I see this logic uncalled for. Ideally the API path should be changed to pluralize if there are multiple tunein result items else ember data would automatically take this path.

Reply:
If I am not using this condition then the call is made to rest/tuneins and as a result HTTP 404 error is encountered.

Reply:
I get that, I was just suggesting if the API's can change.